src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import { Meta } from "@storybook/react/types-6-0";
+import { Story } from "@storybook/react";
+import {Checkbox, CheckboxProps} from "./Checkbox";
+
+export default {
+  title: "Components/Checkbox",
+  component: Checkbox,
+} as Meta;
+
+const Template: Story<CheckboxProps> = (args) => {
+  const [checked, setChecked] = useState(false)
+  return (
+    <div style={{width: "100%"}}>
+      <Checkbox active={checked} handleClick={() => setChecked(!checked)} color={args.color}/>
```

Comment:
I need color to be passed in because the tick is on top of the box, so I can't make it transparent to see the background color behind the box, otherwise it would be white because of the box.